app/presenters/nsm/cost_summary/base.rb

```diff
@@ -30,6 +30,11 @@ def footer_row
           { text: total_cost_cell, classes: 'govuk-table__cell--numeric govuk-summary-list__value-bold' }
         ].compact
       end
+
+      # override if route key for change url does not match key of summary
+      def change_key
```

Comment:
Rather than doing this you can just call the function like `.try(:change_key)` without a dead base member

Reply:
true - I wonder if having the dead base member makes it easier to understand the intention of the method though? or maybe that indicates the method name needs changing

Reply:
I don't know if it does make it easier to understand, seems like it'll rarely need to be overriden (since it hasn't needed to be before now) but it's not a major deal so up to you

Reply:
yeah I think you're right - the intention is already in the code and there's an example of it with a test associated to it